src/main/java/com/revature/byteshare/favorites/favoriteController.java

```diff
@@ -0,0 +1,80 @@
+package com.revature.byteshare.favorites;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping("Favorite")
+public class favoriteController {
```

Comment:
Class should be Pascal Case, like FavoriteController, follow this format for all files